.github/workflows/CD.yml

```diff
@@ -0,0 +1,43 @@
+name: CD
```

Comment:
Unfortunately, the name CD doesn't, from my point of view, bring clarity to the workflow. As I can see, this pipeline only deploys a bookinfo. So, what if we will rename it as cd_bookinfo.yaml or deployment_bookinfo.yaml. What do you think?

Reply:
Btw, the same for the CI